docs/topics/i18n/translation.txt

```diff
@@ -2105,9 +2105,14 @@ message file specific to the project you are composing. The choice is yours.
 All message file repositories are structured the same way. They are:
 
 * All paths listed in :setting:`LOCALE_PATHS` in your settings file are
-  searched for ``<language>/LC_MESSAGES/django.(po|mo)``
-* ``$APPPATH/locale/<language>/LC_MESSAGES/django.(po|mo)``
-* ``$PYTHONPATH/django/conf/locale/<language>/LC_MESSAGES/django.(po|mo)``
+  searched for ``<language>/LC_MESSAGES/<domain>.(po|mo)``
+* ``$APPPATH/locale/<language>/LC_MESSAGES/<domain>.(po|mo)``
+* ``$PYTHONPATH/django/conf/locale/<language>/LC_MESSAGES/<domain>.(po|mo)``
+
+By default, these locations are searched for ``django.(po|mo)`` files.
+You can also specify what translation file names to be searched for
+by adding a ``(path, domain)`` tuple to the :setting:`LOCALE_PATHS` setting.
+:setting:`INSTALLED_APPS` always default to the ``django`` domain.
```

Comment:
I'm not sure this last sentence makes sense - what is it trying to say?

Reply:
What I mean to say is that directories defined inside INSTALLED_APPS will always default to the django domain.
I've changed the message it should be more clear now, if not let me know.